source/release-notes/5.0-compatibility.txt

```diff
@@ -61,6 +61,15 @@ output the ``cache pressure percentage threshold`` and the
 ``current cache pressure percentage`` under
 ``wiredTiger.snapshot-window-settings``.
 
+Shell Changes
+-------------
+
+The ``mongo`` shell has been deprecated in MongoDB v5.0. The
+replacement shell is :mongosh:`mongosh </>`.
+
+The new shell is not packaged with MongoDB. To download and install
```

Comment:
It's not entirely true that the new shell is not packaged with MongoDB.
It's true now, but it won't be true for all platforms at the time of GA. I'll send you the link to the scope via Slack.

Essentially our expectations are:
- Windows: mongosh is **not** part of the server MSI
- Server zip/tar.gz: mongosh is **not** included
- Linux PPAs: there will be meta-packages `mongodb-{org,enterprise}` that include `mongosh`
- Homebrew: mongosh will be part of the [`mongodb-community` formula](https://github.com/mongodb/homebrew-brew/blob/master/Formula/mongodb-community.rb)

Reply:
I updated the text to a more neutral pointer to the Install instructions. The changes can be made there as the packing is updated.